kubernetes/kubernetes/src/main/java/org/arquillian/cube/kubernetes/impl/SessionManager.java

```diff
@@ -220,6 +231,15 @@ public void start() {
         }
     }
 
+    private boolean isRunningFromMaven() {
```

Comment:
Do you think is it worth to move this logic into a "util" class to be reused somewhere else? Or for now leave it as is?